scripts/fix-canvas-deps.ts

```diff
@@ -0,0 +1,46 @@
+/**
```

Comment:
This whole file feels like an amazing kludge, and brittle too. Is there really no other way?

Reply:
Unfortunately I could not find one after banging my head for a few days. The fact that we use Electron seem to break some of the dependencies auto detecting certain configuration things.